Dockerfile

```diff
@@ -59,4 +59,4 @@ ENV ZONEINFO=/zoneinfo.zip
 # UI must be built outside docker
 COPY ./ui/build ./ui/build
 
-CMD ["merlin_api"]
+ENTRYPOINT ["sh", "-c", "merlin_api \"$@\"", "--"]
```

Comment:
This needs to be changed in order to allow other command line arguments to be passed (like the `-config` flag).